src/core/server/app/router/client.ts

```diff
@@ -265,6 +272,7 @@ export async function mountClientRoutes(
       cacheDuration: false,
       disableFraming: true,
       entrypointLoader: manifestLoader.createEntrypointLoader("auth"),
+      templateVariables: { title: "Log In" },
```

Comment:
How do we feel about making this `Sign in` to match our stream button language?

Reply:
✅ !